helm_deploy/hmpps-book-a-video-link-api/values.yaml

```diff
@@ -36,6 +37,8 @@ generic-service:
       DB_NAME: "database_name"
       DB_USER: "database_username"
       DB_PASS: "database_password"
+    gov-notify:
+      NOTIFY_API_KEY: "notify_api_key"
 
```

Comment:
Thinking we'd add the trial key here (for now)? Then later - this can be overridden in PROD with another API key which can send to anyone?

Reply:
Yarp